docs/core/reference/changelog.md

```diff
@@ -4,12 +4,26 @@ This changelog covers updates to cored, the Chain Core server binary.
 
 For updates to subpackages, see below:
 
-- [Mac installer](https://github.com/chain/chain/blob/1.2-stable/installer/mac/CHANGELOG.md)
-- [Windows installer](https://github.com/chain/chain/blob/1.2-stable/installer/windows/CHANGELOG.md)
+- [Mac installer](https://github.com/chain/chain/blob/1.2-stable/desktop/mac/CHANGELOG.md)
+- [Windows installer](https://github.com/chain/chain/blob/1.2-stable/desktop/windows/CHANGELOG.md)
 - [Java SDK](https://github.com/chain/chain/blob/1.2-stable/sdk/java/CHANGELOG.md)
 - [Node.js SDK](https://github.com/chain/chain/blob/1.2-stable/sdk/node/CHANGELOG.md)
 - [Ruby SDK](https://github.com/chain/chain/blob/1.2-stable/sdk/ruby/CHANGELOG.md)
 
+<a name="1.2.1"></a>
+## 1.2.1 (June 13, 2017)
+
+### Major changes
+
+TLS version 1.2 is now required for all HTTPS connections ([#1314](https://github.com/chain/chain/pull/1314)).
```

Comment:
Is there any guidance we can provide here on how to use TLS 1.2 if manual steps are necessary? It seems like at least Java 7 requires some kind of flag to be set for TLS 1.2 to be available on the client side, but I'm not 100% sure.